### PR TITLE
[MIG] [13.0] mgmtsystem_health_safety

### PR DESCRIPTION
**Pull request by flachica**
@NachoAlesLopez @max3903 Please, review

### CODE REVIEW

**Review by dreispt — APPROVED** [review]

**Comment by flachica** [comment]
Depends #299

**Comment by AntoniRomera** [comment]
@flachica Can you review the PR as the depend module is merge?

**Review by FernandoRomera — APPROVED** [review]

**Comment by FernandoRomera** [comment]
@dreispt can you remove the pending review?

File: mgmtsystem_health_safety/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Health and Safety Management System",
+    "version": "13.0.1.0.0",
+    "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
```
Comment (ivantodorovich) [inline mgmtsystem_health_safety/__manifest__.py]: https://github.com/OCA/management-system

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: mgmtsystem_health_safety/__manifest__.py (hunk 1 above)
Comment (ivantodorovich) [reply mgmtsystem_health_safety/__manifest__.py]: > The website key must be https://github.com/OCA/<repo>, so as to provide the most relevant link to discover more information about the addon. That link shows the repository README. Alternatively https://github.com/OCA/<repo>/tree/<branch>/<addon> may be used, to provide a direct link to the addon README, which includes proper credits (authors, contributors and theirs companies), and links to the relevant information on the OCA website.

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id6

File: mgmtsystem_health_safety/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (ivantodorovich) [inline mgmtsystem_health_safety/__manifest__.py]:
```suggestion
# © 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Use short version

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by ivantodorovich** [comment]
@flachica there's only a quick pre-commit thingy to fix, and we're all set :)

**Pull request merged**